bpm/src/main/java/org/jboss/pnc/bpm/causeway/BuildResultPushManager.java

```diff
@@ -338,24 +343,26 @@ private Build getNpmBuild(
 
     private void addLogs(BuildRecord buildRecord, Set<Logfile> logs) {
         String externalBuildID = buildRecord.getId().toString();
-        if (buildRecord.getBuildLogSize() != null && buildRecord.getBuildLogSize() > 0) {
+        long buildLogSize = bifrost.getFinalLogSize(buildRecord.getId().toString(), "build-log");
+        if (buildLogSize > 0) {
             logs.add(
                     Logfile.builder()
                             .filename("build.log")
                             .deployPath(getBuildLogPath(externalBuildID))
-                            .size(buildRecord.getBuildLogSize())
-                            .md5(buildRecord.getBuildLogMd5())
+                            .size((int) buildLogSize)
+                            .md5(buildRecord.getbuildOutputChecksum())
                             .build());
         } else {
             logger.warn("Missing build log for BR.id: {}.", externalBuildID);
         }
-        if (buildRecord.getRepourLogSize() != null && buildRecord.getRepourLogSize() > 0) {
+        long alignmentLogSize = bifrost.getFinalLogSize(buildRecord.getId().toString(), "alignment-log");
```

Comment:
the processContext for build logs should be `build-<<buildId>>` right?

Reply:
I belive it's only buildId, right @thescouser89 ?

Reply:
Answered in the previous question.